lib/screens/home.dart

```diff
@@ -76,20 +76,20 @@ class HomeState extends State<Home> with SingleTickerProviderStateMixin {
       _isLoading = true;
     });
 
+    final appName = await getAppName();
     try {
-      const filePath = 'encryption/enc-keys.ttl';
+      final filePath = '$appName/encryption/enc-keys.ttl';
```

Comment:
It seems that $appName will be the location for all data files for the app. Do we need to specify it every time? Can `solidpod` itself know what the basename is.

Also, Why do I as the app developer need to know this path to the enc-keys.ttl? Can that be done under the bonnet?

Reply:
A function to get the encryption key file path is now provided in `solidpod`. If this function should be exported could be another discussion.